docs/docs/how-tos/nebari-gcp.md

```diff
@@ -0,0 +1,155 @@
+- In the drop down menu, select "Create new key";
+- Leave the "Key Type" as `JSON`;
+- Click "Create" to create the key and save the key file to your system.
+
+Store this credentials file in a well known location and make sure to set yourself exclusive permissions. You can change the file permissions by running the command `chmod 600 <filename>` on your terminal.
+
+:::warning Warning
+The service account key file provides access to your GCP project. It should be treated like any other secret credentials. Specifically, it should never be checked into source control.
+:::
+
+By default, Nebari will try to use the credentials associated with the current Google Cloud infrastructure/environment for authentication. Provide authentication credentials to Nebari by setting the following environment variables:
+
+```bash
+export GOOGLE_CREDENTIALS="path/to/JSON/file/with/credentials"
+export PROJECT_ID="projectIDName"
+```
+The **project ID** information can be found at the Google Console homepage, under **Project info**.
+
+This variables applies only to your current shell session. If you want the variable to apply to future shell sessions, set the variable in your shell startup file, for example in the `~/.bashrc` or `~/.profile` file.
+
+
+:::note
+The following steps assumes you have completed the [Install Nebari](/started/installing-nebari) section, has confirmed that `nebari` is successfully installed in your environment and opted for **GCP** as your cloud provider as well as installed and initialized `gcloud`. If you had any issues during the installation, please visit our getting started [troubleshooting](/started/troubleshooting) section for extra advice.
+:::
+
+## Nebari Initialize
+
+Great, you’ve gone through the [Nebari Installation](/started/installing-nebari.md) and [Authentication setup](#authentication) steps, and have ensured that all the necessary environment variables have been properly set. It is time to initialize and deploy Nebari!
+
+In your terminal, we advise you to start by creating a new project folder. Here, we will name the new folder as `data`:
+
+```bash
+mkdir data && cd data
+```
+When you first initialize Nebari, you will be creating a `nebari-config.yaml` that contains a collection of preferences and settings for your deployment. The command bellow will generate a basic config file with an infrastructure based on **GCP**, named **projectname**, where the domain will be **domain** and the authetication mode set to **password**.
+
+```bash
+nebari init gcp --project projectname --domain domain --auth-provider password
+```
+:::note Note
+You will be prompted to enter values for some of the choices above if they are omitted as command line arguments (for example project name and domain)
+:::
+
+Once `nebari init` is executed, you should then be able to see the following output:
+```bash
+Securely generated default random password=*** for Keycloak root user
+stored at path=/tmp/NEBARI_DEFAULT_PASSWORD
```

Comment:
Can we add a small note that the `tmp` folder on MacOS is found somewhere like: `path=/var/folders/xx/xxxxx/T/QHUB_DEFAULT_PASSWORD`

Reply:
yup, thanks for pointing it out.

Reply:
Looks good to me!